ajax/libs/react/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react",
-  "filename": "react.min.js",
-  "version": "15.6.1",
+  "filename": "umd/react.production.min.js",
```

Comment:
Please leave the filename alone here as it's not changed in the latest stable version yet.